setup.py

```diff
@@ -7,36 +7,37 @@
 extras_require = {
     'dexterity': [
         'plone.app.dexterity',
-        ]}
+        ],
+    'plone4': [
```

Comment:
Can we change the extras name to `tests_plone4`?
- extras makes only sense when testing ftw.publisher.core itself with Plone 4
- matches the same extras in ftw.publisher.receiver

Reply:
@Nachtalb has changed that.